dev_guide/builds/triggering_builds.adoc

```diff
@@ -436,6 +436,21 @@ strategy:
 This ensures that the triggered build uses the new image that was just pushed to
 the repository, and the build can be re-run any time with the same inputs.
 
+An image change trigger can be paused allowing for multiple changes on the referenced
```

Comment:
One small suggestion for consistency, 

s/An image change trigger can be paused allowing.../You can pause an image change trigger allowing...

Otherwise LGTM :stars:

Reply:
thanks, reworded as you suggested